SetupDataPkg/Tools/ConfigEditor.py

```diff
@@ -390,14 +398,22 @@ def __init__(self, master=None):
         tkinter.Frame.__init__(self, master, borderwidth=2)
 
         self.menu_string = [
-            "Save Config Data to Binary",
-            "Load Config Data from Binary",
-            "Load Config Changes from Delta File",
-            "Save Full Config Data to SVD File",
-            "Save Config Changes to SVD File",
-            "Load Config Data from SVD File",
-            "Save Config Changes to Delta File",
-            "Save Full Config Data to Delta File",
+            "Save Config Data to Var List Binary",
+            "Load Config Data from Var List Binary",
+            'Update Config from Change File',
+            'Save Full Config Data to SVD File',
+            'Save Config Changes to SVD File',
+            'Load Config Data from SVD File',
```

Comment:
I guess this should be `Load Config Changes **`?

Reply:
Sure, I'll unify the language.